tests/models/test_decam.py

```diff
@@ -362,7 +362,7 @@ def test_decam_download_and_commit_exposure(
                 match = re.search( r'^c4d_(?P<yymmdd>\d{6})_(?P<hhmmss>\d{6})_ori.fits', fname )
                 assert match is not None
                 # Todo : add the subdirectory to dbfname once that is implemented
-                dbfname = ( f'c4d_20{match.group("yymmdd")}_{match.group("hhmmss")}_{exposure.filter[0]}_'
+                dbfname = ( f'c4d_20{match.group("yymmdd")}_{match.group("hhmmss")}_{exposure.filter}_'
```

Comment:
This right here is one of the reasons why I really like going to short filter names as the standard.  Probably, usually, in the old world, the short filter name would have been the first character of the long filter name... but could we coun't on that?  We had that baked in, but, now, don't have to worry about it.

Reply:
Yes, I noticed that here too! Much more flexible this way.